agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4321,4 +4321,4 @@ bool slave_thread::send_cmdu_to_controller(ieee1905_1::CmduMessageTx &cmdu_tx)
     }
     return message_com::send_cmdu(master_socket, cmdu_tx, backhaul_params.controller_bridge_mac,
                                   backhaul_params.bridge_mac);
-}
\ No newline at end of file
+}
```

Comment:
Is this just a new line at the end of the file?

Reply:
yes